### PR TITLE
Options update

### PR DESCRIPTION
**Pull request by swflint**
Create User-settable options and enable both new and old-style capture Links

### CODE REVIEW

**Comment by swflint** [comment]
Supercedes #27 and #25.

**Comment by sprig** [comment]
Thanks for your contribution!

I am more inclined to accept this PR rather than #30. 
However, please change default template tags that they are consistent with current ones. Likewise, I would like the new style links to only apply (by default) to new installations rather than updates to old ones.

In summary, if I already have the extension installed and I update to the new version, everything should continue working without my intervention.

**Comment by swflint** [comment]
If I may ask, why would you prefer this over #30?  #30 continues this work, but adds further functionality, making the extension more useful.

I will certainly change the defaults though.

File: capture.js [hunk 1]
```diff
@@ -1,26 +1,38 @@
...
+                                   "[)]", escape(")")),
+                       "[']" ,escape("'"));
+}
 
-    if (selection != "") { uri += '/' + esc(selection); };
+function createCaptureURL(template, title, url, selection, oldStyle) {
+    if (oldStyle == true)
+        return "org-protocol://capture:/"+template+'/'+url+'/'+title + ((selection === '') ? '' : ('/' + selection));
+    else
+        return "org-protocol://capture?template="+template+'&url='+url+'&title='+title+((selection === '') ? '' : ('&body=' + selection));
```
Comment (sprig) [inline capture.js]: Since you are using a regular URL syntax here, the selection should be escaped using encodeURIComponent as well.

**Pull request merged**